azext_iot/common/embedded_cli.py

```diff
@@ -69,6 +69,9 @@ def success(self) -> bool:
         logger.debug("Operation error code: %s", self.error_code)
         return self.error_code == 0
 
+    def get_error(self) -> Exception:
+        return self.az_cli.result.error
```

Comment:
![image](https://user-images.githubusercontent.com/73560279/210425853-6218beba-fd09-424e-813a-520660069699.png)
yup, it will be empty if there is no error